tests/dedupe_unit_test.py

```diff
@@ -33,25 +33,50 @@ def check_nb_duplicates(self, expected_number_of_duplicates):
             trs = driver.find_elements_by_xpath('//*[@id="open_findings"]/tbody/tr')
             for row in trs:
                 concatRow = ' '.join([td.text for td in row.find_elements_by_xpath(".//td")])
+                # print(concatRow)
                 if '(DUPE)' and 'Duplicate' in concatRow:
                     dupe_count += 1
 
             if (dupe_count != expected_number_of_duplicates):
-                print("duplicate count mismatch, let's wait a bit for the celery dedupe task to finish and try again (5s)")
+                print("duplicate count mismatch, let's wait a bit for the celery dedupe task to finish and try again (15s)")
```

Comment:
There is a small type, waiting for 5s now.

Reply:
ty fixed